### PR TITLE
[FLINK-34721][Test]Correct BarrierAlignmentUtilTest Exception explanation

### PR DESCRIPTION
**Pull request by HCTommy**
This pull request is to fix a grammar error in comments.

## Brief change log

Fixed a grammar error in file  BarrierAlignmentUtilTest.test

## Verifying this change

This change fixed a grammar in comments.  'should not do', not 'should not done'.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? ( no)

### CODE REVIEW

**Comment by HCTommy** [comment]
@RyanSkraba Hi, I fixed a grammar error in the explanation of a thrown error when testing the barrier. Could you please review in your available time.

**Review by RyanSkraba — APPROVED** [review]
I guess **_technically_** this looks good to me, but I would recommend in the future that you avoid one-line commits for typo PRs.  It's very costly in terms of committer attention as well as CI build resources...

My personal opinion is that fixing typos and grammar _is_ valuable and gives a better impression of a high-quality project, but unless they are **very wrong** and **very visible**, you should save them to submit a batch of typo fixes in a single PR.  The more typo fixes you can do, the more compelling it is to attract committer attention and get it merged.

I'll leave it to an actual Flink committer to determine how to proceed with this one-liner, noting that this has already been [addressed](https://github.com/apache/flink/pull/24476#issuecomment-1997312069).

**Pull request closed without merging**
